### DISCUSSION TITLE
What's the latest version?

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [Sketchware-Pro/Sketchware-Pro#857] [Q&A] [answered]
The website says the app is outdated. I have the version: v6.4.0-rc05-minApi26

If there is another version superior to this send me.

Thanks.

**Comment by DevVigilante** [comment]
Which website you visited?
If @NiceSapien website then it shows wrong version.

**Reply by Jeiel0rbit** [reply]
https://sketchware-pro.ga/

**Reply by DevVigilante** [reply]
> https://sketchware-pro.ga/

This website needs to be updated.

**Comment by DevVigilante** [comment, marked as answer]
Sketchware Pro v6.4.0 test build 05 is latest.

**Comment by NiceSapien** [comment]
My bad! The link is latest, however, I forgot to change the link text. Will do that now.

**Comment by NiceSapien** [comment]
@Jetrom17 done. [View it here.](https://sketchware-pro.ga/download)
